pkg/sources/github/github.go

```diff
@@ -850,25 +856,33 @@ func (s *Source) handleRateLimit(errIn error, res *github.Response) bool {
 	// https://docs.github.com/en/rest/overview/resources-in-the-rest-api#exceeding-the-rate-limit
 	if retryAfter <= 0 && res != nil {
 		var err error
-		remaining, err = strconv.Atoi(res.Header.Get("x-ratelimit-remaining"))
-		if err != nil {
-			knownWait = false
+
+		remainingHeader := res.Header.Get("x-ratelimit-remaining")
+		if remainingHeader != "" {
+			if remaining, err = strconv.Atoi(remainingHeader); err != nil {
```

Comment:
If the parsing of this header (or the other one) fails, do you think it's worth logging out the unparseable value?

Reply:
I don't think so. The only failure I've encountered in testing is when the header is empty (`""`), which should no longer be an issue.